src/redux/reducers/cacheReducer.ts

```
@@ -301,6 +309,24 @@ const cacheReducer = (state = initialState, action) => {
         ...state, // spread operator in requried here, otherwise persist do not register change
       };
 
+    case UPDATE_SPOTLIGHT_META:
+      if (!state.spotlightMeta) {
+        state.spotlightMeta = {};
+      }
+
+      const _alreadyDismissed = state.spotlightMeta[payload.id]?.dismissed || false;
```

Biome (2.4.16)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)